### ISSUE TITLE
Installation problems

### ISSUE THREAD

**Issue by CarlVerret** [EgorBo/Disasmo#23]
Hi,
Trying to install on Vs2019 enterprise 16.10.4, setup is always complaning about

**Comment by xtqqczze** [comment]
I'm seeing the same issue attempting to install for Visual Studio 2019 Community 16.11.1.

[dd_VSIXInstaller_20210822120453_1cbc.log](https://github.com/EgorBo/Disasmo/files/7027216/dd_VSIXInstaller_20210822120453_1cbc.log)

**Comment by MithrilMan** [comment]
I've the same problem.
For the record, I've also VS 2022 Preview 3 installed, just saying in case "it's not a coincidence"

**Comment by Blokyk** [comment]
Yep, exactly the same here, been looking all around the internet but can't find anything :/ VS2019 Community + VS2022 Preview 3

**Comment by omariom** [comment]
+1
